### PR TITLE
Update Showcase Hub Handbook Authoring image tag

### PR DESCRIPTION
**Pull request by jnywong**
I downgraded `docutils` to `v0.17.1` as advised in the [Jupyter Book docs](https://jupyterbook.org/en/stable/content/citations.html?highlight=docutils#citations-and-bibliographies) (see PR https://github.com/2i2c-org/community-showcase/pull/49) so that Citations and Bibliographies block would work properly in the Handbook Authoring image available on the Showcase Hub. 

This pull request changes the config file to deploy the updated image hosted on quay.io with the new tag.

Question: is this the correct process for all future changes to the profileList on the Showcase Hub? Thanks in advance!

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]
> Question: is this the correct process for all future changes to the profileList on the Showcase Hub?

Yes! Also, these image bump type PRs are [exempt from our code review process](https://infrastructure.2i2c.org/contributing/code-review/#self-merging-as-a-2i2c-engineer) so feel free not wait for a review in the future :)

**Comment by jnywong** [comment]
@sgibson91 Cheers! I'll wait until the CI is sorted (@GeorgianaElena is looking at it in PR #3611 https://github.com/2i2c-org/infrastructure/pull/3611#issuecomment-1888884957) before self-merging :)

Also, I think we need to sort my access to this repo since I had to make a fork.

**Comment by sgibson91** [comment]
That test is just checking that the YAML is valid, and I can tell by the diff that it would pass, so I'll merge it.

Yeah, I think because you're the first of a new team, there isn't a corresponding GitHub team to add you to to get you the appropriate access (e.g. if you were in engineering, you'd be added that team and you'd have the right access), so probably something to bring up with @jmunroe. That being said, I think we all follow the forking process anyway, so this is more about write access.

**Comment by jmunroe** [comment]
There is already a @2i2c-org/partnerships-and-community-guidance github team (which @jnywong is a member).  That team had only granted 'triage' access rights to `2i2c-org/infrastructure` .  I've changed the permissions of the team so that membership in @2i2c-org/partnerships-and-community-guidance now grants 'maintain' rights to this repo so @jnywong should be able to self-merge when needed.

**Comment by sgibson91** [comment]
@jmunroe it's probably a good idea to also add that team to the docs repo regarding https://github.com/2i2c-org/docs/pull/204#issuecomment-1887529491

**Comment by jmunroe** [comment]
> @jmunroe it's probably a good idea to also add that team to the docs repo regarding https://github.com/2i2c-org/docs/pull/204#issuecomment-1887529491

Yep! I added the docs repo and the 2i2c website repo as well

**Comment by jmunroe** [comment]
> @jmunroe it's probably a good idea to also add that team to the docs repo regarding https://github.com/2i2c-org/docs/pull/204#issuecomment-1887529491

Yep! I added the docs repo and the 2i2c website repo as well

**Pull request merged**
